omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -348,6 +350,10 @@ Client Details:
     # the formerowner from group, just unsets the ownership. This works fine
     # with Create and Edit Group privilege only
     $ bin/omero user leavegroup testgroup --name formerowner  --as-owner
+    # Move a Dataset to a group where the prospective owner is member
+    # and include all annotations on objects
+    # linked to that Dataset even if they do not belong to the same user
+    $ bin/omero chgrp 5 Dataset:51 --include Annotation
     # Transfer ownership to user 55 of the whole hierarchy of the Project 112
     # including the links, as long as all the objects and links in the
     # hierarchy belong to the same user
```

Comment:
@mtbc : Thank you. Sorry for the confusion, I shoiuld have not asked in such complicated way. I had 2 questions, the first one concerning the ``chown`` (you thought I am asking about ``chgrp`` which confused you, sorry about it). Only my second question was about ``chgrp``